crypto/merkle/proof_op.go

```diff
@@ -68,6 +68,33 @@ func (poz ProofOperators) Verify(root []byte, keypath string, args [][]byte) (er
 	return nil
 }
 
+func (poz ProofOperators) VerifyKeys(root []byte, keys [][]byte, args [][]byte) (err error) {
+	for i, op := range poz {
+		key := op.GetKey()
+		if len(key) != 0 {
```

Comment:
when would `len(key)==0`?

Reply:
that's a good question, so afaict, the only time this would be zero is if one of the keys that the user passes it is 0. 

this was blindly copy pasted from the original method https://github.com/celestiaorg/celestia-core/blob/475e042d783331f939ffeb6a10fbfe7113effde0/crypto/merkle/proof_op.go#L47 just to change the api, but I can see how with the api change, this might not be the best place to check for this.

do you think we should still handle this here? or should we stick with as close to the original as possible?